src/foremast/awslambda/awslambda.py

```diff
@@ -61,6 +61,22 @@ def _check_lambda(self):
             pass
         return exists
 
+    def _check_lambda_alias(self):
```

Comment:
This function looks oddly similar to `foremast.utils.awslambda.get_lambda_alias_arn`. Can we refactor down to one function?